README.md

```diff
@@ -1,97 +1,330 @@
+* [Considerations for cross-program contexts](#considerations-for-cross-program-contexts)
+* [Closing](#closing)
+* [Provide Feedback](#provide-feedback)
+
+## Introduction
+
+Eligible people, who have the most need for healthcare, cash assistance, and food assistance are struggling to receive or renew benefits. This struggle is in part due to the burden of income verification. The burden of proof falls on them, not the government service.
+
+<div>
+The Income Reporting Playbook offers guidance and templates to improve the <div class="tooltip">client<span class="tooltiptext">“Client” refers to people applying for or renewing a benefit.</span></div> experience of providing income during benefit applications and renewals.
+</div><br/>
+In this playbook, you’ll find:
+
+* Recommended best practices
+* Common terms and plain language definitions
+* Example language and visual templates for income
+
+Following this guidance can improve application form accessibility, equity, and usability.
+
+**Who is this playbook for?**
+
+This playbook is a guide for:
+
+* Benefits program administrators and their staff
+* Other state, county, and local officials who support benefit eligibility and enrollment process
+* Vendors and contractors working with agencies to deliver benefits to clients
+
+This guide is not intended to be used by benefit clients.
+
+**How can I use the information in this guide?**
+
+* Identifying changes to make to your application/renewal forms
+* Identifying areas of your application/renewal forms to test with clients
+
+This guide is primarily intended to help improve online application/renewal forms.
+
+## What is plain language?
+
+**Plain language** (also called plain writing or plain English) is communication your audience can understand the first time they read or hear it.
+
+The <a href="https://www.govinfo.gov/app/details/PLAW-111publ274">Plain Writing Act of 2010</a> defines plain language as "writing that is clear, concise, well-organized, and follows other best practices appropriate to the subject or field and intended audience."
+
+Plain language helps your audience:
+
+* Find what they need
+* Understand what they find the first time they read or hear it
+* Use what they find to meet their needs
+
+Plain language informs content, but it also informs the organization and layout of content. For income verification, this means that there is opportunity to:
+
+* Refine the written content to support better income reporting
+* Refine the organization of the content and sections to promote clarity and prevent context switching when reporting income
+* Refine the designs so that the visualization of the content enables clear and simple income reporting
+
+Writing in plain language also improves the output of automated multi-language support tools. However, plain language and automated translations are not a replacement for human translation services.
+
+## Impact of plain language implementations
```

Comment:
It just _feels_ like there's too little vertical space above new secondary headers. The sections just seem smushed together. Might be my personal preference though.

<img width="194" alt="image" src="https://github.com/navapbc/FFS-IncomeReportingToolkit/assets/776667/2e4d9250-8d49-4097-82ff-61f5411919c9">